lib/puppet/util/fips.rb

```diff
@@ -0,0 +1,28 @@
+require 'puppet/util/platform'
+# Method to determine whether where are running on a FIPs enabled host
```

Comment:
Since this is a platform-y thing, like `windows?`, let's just add the `fips_enabled?` method to `Puppet::Util::Platform`.